### PR TITLE
Use new avatar subdomain

### PR DESCRIPTION
**Pull request by nicl**
The Avatar API now lives on avatar.theguardian.com. This means we can use it and stop using the deprecated GU_U cookie for authentication.

### CODE REVIEW

**Comment by nicl** [comment]
cc @guardian/identity @guardian/discussion

File: static/src/javascripts/projects/common/modules/discussion/user-avatars.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 define([
     'common/utils/$',
     'bonzo',
-    'common/modules/discussion/api'
-], function ($, bonzo, discussionApi) {
+    'common/modules/avatar/api'
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/discussion/user-avatars.js]: why `/api` and not just `avatar`?

Comment (sndrs) [inline static/src/javascripts/projects/common/modules/discussion/user-avatars.js]: should it also go in `modules/identity/avatar`?

File: static/src/javascripts/projects/common/modules/discussion/user-avatars.js [hunk 2]
```diff
@@ -13,14 +13,21 @@ define([
             updating = bonzo(bonzo.create('<div class="is-updating"></div>')),
             avatar = bonzo(bonzo.create('<img class="user-avatar__image" alt="" />')),
             userId = container.data('userid');
+
         container
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/discussion/user-avatars.js]: all dom touching should be done inside `fastdom`

File: static/src/javascripts/projects/common/modules/discussion/user-avatars.js (hunk 1 above)
Comment (nicl) [inline static/src/javascripts/projects/common/modules/discussion/user-avatars.js]: I think it deserves a separate folder as it is potentially useful for Discussion as well as Identity code/features.

Re: avatar versus api, no strong preference, was just following what other modules do here - for example discussion/api.js.

File: static/src/javascripts/projects/common/modules/discussion/user-avatars.js (hunk 2 above)
Comment (nicl) [inline static/src/javascripts/projects/common/modules/discussion/user-avatars.js]: @sndrs thanks. How important is this here? The code is written to rely on the return value of the bonzo create operations, whereas fastdom seems to return simply an op identifier. The profile page this impacts has hardly any dom elements on it (https://profile.theguardian.com/public/edit) and I'm a bit reluctant to start rewriting existing code here unless absolutely necessary.

Comment (sndrs) [inline static/src/javascripts/projects/common/modules/discussion/user-avatars.js]: sorry - this isn't part of this PR, should have looked closer

**Comment by sndrs** [comment]
it'd be good to fastdom all these dom read/writes at some point, but :+1: for this

**Pull request merged**